new-pipeline/dist.html

```diff
@@ -23,185 +23,266 @@
         </header>
-        <div class="row">
-            <div class="col-md-12">
-                <h2 style="font-size: 18px; margin-bottom: 0">
-                    <span style="font-weight: bold">
-                        <select id="measure" class="multiselect" title="Value being measured"></select>
-                        distribution for
-                        <select id="filter-product" class="multiselect" title="Product name" data-all-selected="all products" data-n-selected="products" multiple></select>
-                        <select id="channel-version" class="multiselect" title="Build version"></select>,
-                    </span>
-                    on
-                    <select id="filter-os" class="multiselect" title="Target OS" data-all-selected="any OS" data-n-selected="OSs" multiple></select>
-                    <select id="filter-arch" class="multiselect" title="Product architecture" data-all-selected="any architecture" data-n-selected="architectures" multiple></select>
-                    with
-                    <select id="filter-process-type" class="multiselect" title="Process type" data-all-selected="any process type"></select>
-                    and compare by
-                    <select id="compare" class="multiselect" title="Filter name to compare options for">
-                        <option value="">none</option>
-                        <option value="os">target OS</option>
-                        <option value="osVersion">target OS version</option>
-                        <option value="architecture">product architecture</option>
-                        <option value="child">process type</option>
-                    </select>
-                </h2>
-                <hr></hr>
-                <figure class="col-md-12">
-                  <figcaption id="dist-caption"></figcaption>
-                    <div class="col-md-9" id="plots">
-                        <div class="row">
-                            <div class="col-md-6" style="text-align: center">
-                                <h3><select id="selected-key1" class="multiselect" title="Selected key"></select></h3>
-                                <div style="margin-bottom: 20px; z-index: -1">
-                                    <div id="distribution1"></div>
+        <ul class="nav nav-tabs">
+            <li class="active" ><a href="#dist" data-toggle="tab"><strong>Distribution</strong></a></li>
+            <li><a data-toggle="tab" href="#evo"><strong>Evolution</strong></a></li>
+        </ul>
+        <div class="tab-content">
+            <div id="dist" class="tab-pane fade in active">
+                <div class="row">
+                    <div class="col-md-12">
+                        <h2 style="font-size: 18px; margin-bottom: 0">
+                            <span style="font-weight: bold">
+                                <select id="measure" class="multiselect" title="Value being measured"></select>
```

Comment:
`measure` is a duplicated id. ids must be unique in the document, otherwise the first one wins.

There are other duplicated ids.